.pre-commit-config.yaml

```diff
@@ -10,12 +10,13 @@ repos:
       - id: codespell
         args:
           - --ignore-words-list=nd,reacher,thist,ths
-  - repo: https://gitlab.com/pycqa/flake8
+  - repo: https://gitlab.com/PyCQA/flake8
     rev: 3.9.2
     hooks:
       - id: flake8
         args:
-          - --ignore=E203,E402,E712,E722,E731,E741,F401,F403,F405,F524,F841,W503,E302,E704
+          - '--per-file-ignores=*/__init__.py:F401 gym/envs/registration.py:E704 E302'
+          - --ignore=E203,W503,E741
```

Comment:
I hadn't considered that, good idea. Personally, I think it is better to just fix the error so we have as minimum `noqa` comments as possible. @RedTachyon Do you know the meaning of these two variables?

Reply:
Agree, the minimum `# noqa` the better, and it'd be better fixing the issues, I'm just saying I'd remove the globally ignored E741 rule so that Flake8 will be able to catch too short variables names on other part of the codebase in the future.

Reply:
Will be fixed in the next PR